src/app/(pages)/page.tsx

```diff
@@ -1,9 +1,164 @@
+      >
+        <h1 className=" text-4xl font-bold">Upcoming Events</h1>
+        <div>
+          {events.map((item) => {
+            return (
+              <EventCard
+                heading={item.heading}
+                key={item.id}
+                img={item.img}
+                desc={item.desc}
+              />
+            );
+          })}
+        </div>
+        <Line />
+      </section>
+
+      <section id="projects" className=" w-10/12 mx-auto flex flex-col gap-10">
+        <h1 className=" text-4xl font-bold mx-auto">Projects</h1>
+        <div>
+          <ProjectCarausal />
+        </div>
+        <div className=" mx-auto">
+          <Button text="View More" rounded={true} linkTo="/" />
+        </div>
+        <Line />
+      </section>
+
+      <section id="aboutCollege" className=" w-11/12 mx-auto ">
+        <h1 className=" font-bold text-4xl">About college</h1>
+        <div className="flex gap-10">
+          <Image src={photo} alt="" />
+          <div className="flex flex-col gap-10">
+            <h2 className=" text-3xl font-semibold">
+              Shri Vaishnav Vidyapeeth Vishwavidyalaya
+            </h2>
+            <p>
+              Shri Vaishnav Vidyapeeth Trust believes in taking the nation
+              forward by improving the quality of life of its citizens by
+              continuously working in the sphere of education, health and
+              environment.To create an educational environment that engages deep
+              intellectual moral and spiritual stimulation there by nurturing
+              leadership. To impact learning through understanding knowledge
+              enrichment skill development and positive attitude formation.To
+              encourage innovation thinking with self discipline & social
+              responsibility.
+            </p>
+          </div>
+        </div>
+        <Line />
+      </section>
+
+      <section id="vcHod" className=" w-11/12 mx-auto ">
+        <h1 className=" text-4xl font-bold">VC & HOD</h1>
+        <div className="flex gap-5">
+          {collegeManagement.map((person) => (
+            <FacultyCard key={person.id} person={person} />
+          ))}
+        </div>
+        <Line />
+      </section>
+
+      <section id="faculty" className=" w-11/12 mx-auto ">
+        <h1 className=" text-4xl font-bold">Faculty coordinator and mentor</h1>
+        <div className="flex gap-5">
+          {coordinators.map((person) => (
+            <FacultyCard key={person.id} person={person} />
+          ))}
+        </div>
+        <Line />
+      </section>
+
+      <section id="boardMembers" className=" w-11/12 mx-auto ">
+      <h1 className=" text-4xl font-bold">Faculty coordinator and mentor</h1>
+        <div className="flex gap-5">
+          {boardMembers.map((person) => (
+            <TeamCard key={person.id} person={person} />
+          ))}
+        </div>
+        <Button linkTo="/team" text="View more" rounded={true}/>
```

Comment:
the above written code is same so why you think there is need to remap these hence refactor code